### PR TITLE
Adding the ability to write tests inline, without fixture files

### PR DESCRIPTION
**Pull request by TheSavior**
Closes https://github.com/facebook/jscodeshift/issues/203

### CODE REVIEW

File: src/testUtils.js [hunk 1]
```diff
@@ -47,25 +66,10 @@ function runTest(dirName, transformName, options, testFilePrefix) {
...
-
-  // Jest resets the module registry after each test, so we need to always get
-  // a fresh copy of jscodeshift on every test run.
-  let jscodeshift = require('./core');
-  if (module.parser) {
-    jscodeshift = jscodeshift.withParser(module.parser);
-  }
-
-  const output = transform(
-    {path: inputPath, source},
```
Comment (TheSavior) [inline src/testUtils.js]: The one thing that feels a bit gross in this change is that inline tests aren't able to have a path here. It seems like most transforms only use source, but people will be unable to use inline tests with their transform if they use the file path directly.

**Comment by fkling** [comment]
cc @Daniel15 who built the test utils

**Comment by Daniel15** [comment]
This seems reasonable to me... I'll take a closer look when I'm at my
computer.

I've also been meaning to change the test util to use Jest's snapshot
feature (keep the existing stuff for backwards compatibility, and use
snapshots when an output file isn't found so it's used for any future
tests).

**Comment by TheSavior** [comment]
Yeah, I was thinking about the snapshot test option as I was writing this. I think the snapshot test piece will be able to use the `runInlineTest` function in this pr under the hood.

File: src/testUtils.js [hunk 2]
```diff
@@ -47,25 +66,10 @@ function runTest(dirName, transformName, options, testFilePrefix) {
   );
   // Assumes transform is one level up from __tests__ directory
   const module = require(path.join(dirName, '..', transformName + '.js'));
-  // Handle ES6 modules using default export for the transform
-  const transform = module.default ? module.default : module;
-
-  // Jest resets the module registry after each test, so we need to always get
```
Comment (Daniel15) [inline src/testUtils.js]: It looks like these comments have gone missing - Could you please re-insert them?

**Comment by Daniel15** [comment]
This looks pretty good to me! Please re-insert the comments that have been deleted:
```
// Handle ES6 modules using default export for the transform
```
```
// Jest resets the module registry after each test, so we need to always get
// a fresh copy of jscodeshift on every test run.
```

And I'll merge this 😃

**Comment by TheSavior** [comment]
Whoops. Good catch. Updated.

**Comment by Daniel15** [comment]
Thank you! :D

**Pull request merged**